aggregator/src/lib.rs

```diff
@@ -6,14 +6,15 @@ use alloy::{
             BlobGasFiller, ChainIdFiller, FillProvider, GasFiller, JoinFill, NonceFiller,
             WalletFiller,
         },
-        Identity, IpcConnect, ProviderBuilder, RootProvider,
+        Identity, IpcConnect, Provider, ProviderBuilder, RootProvider, WalletProvider,
```

Comment:
warning: unused import: `WalletProvider`

```suggestion
        Identity, IpcConnect, Provider, ProviderBuilder, RootProvider,
```

Reply:
[Remove unused import](https://github.com/gizatechxyz/avsthon/pull/11/commits/c1953e9e9817ef6e13cad5bb58770dfaf5ca34f1) apply to the others